applications/vanilla/modules/class.tagmodule.php

```diff
@@ -157,7 +157,7 @@ public function assetTarget() {
      * @return string
      */
     public function inlineDisplay() {
-        if (!c('EnabledPlugins.Tagging')) {
+        if (!c('Vanilla.Tagging.EnableUI')) {
```

Comment:
My focus was bringing the code into core. I did no do any modification that was not needed.

I guess it wouldn't hurt to clean it up a bit though.